src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -589,4 +570,25 @@ public static void assertFileIsReadable(final Path path) {
             throw new UserException.CouldNotReadInputFile(path, cloudBoom.getCode() + ": " + cloudBoom.getMessage(), cloudBoom);
         }
     }
+
+    /**
+     * Checks that one or more user provided files are in fact regular (i.e. not a directory or a special device) readable files.
+     *
+     * @param files the input files to test.
+     * @throws IllegalArgumentException if any input file {@code file} is {@code null} or {@code files} is {@code null}.
+     * @throws UserException if any {@code file} is not a regular file or it cannot be read.
+     */
+    public static void canRead(final File... files) {
```

Comment:
Looks like you already hit merge on this branch, but I neglected to mention that the new name for the method is also not ideal. "IOUtils.canRead(directory)" will fail on a readable directory, since a directory is not a regular file, but the method name implies that it should pass.

Also, as @magicDGS points out, the tests are now misplaced. I recommend that you do a second PR to fix these issues @davidbenjamin